assets/syntaxes/Packages/XML/syntax_test_xml.xml

```diff
@@ -573,23 +573,23 @@
 ##    ^^^^^^ keyword.declaration.entity
 ##           ^ punctuation.definition.entity
 
-    <!ENTITY % entiy-name
+    <!ENTITY % entity-name
```

Comment:
It took me a while to decide that the spelling here didn't matter, but I'm pretty sure it doesn't and thus it should be possible to fix.